File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/ChatMessageRetrieverDTO.java
package com.example.myapplication.dtos;

/**
 * Created by Michal on 18/01/14.
 */
public class ChatMessageRetrieverDTO {
    public int SenderId;
    public int RecipientId;
    public LoadRangeDTO LoadRangeDTO;

    public ChatMessageRetrieverDTO(int senderId, int recipientId, LoadRangeDTO loadRangeDTO)
    {
        this.SenderId = senderId;
        this.RecipientId = recipientId;
        this.LoadRangeDTO = loadRangeDTO;
    }
}

File: Project/WebService/Services/DTOs/ChatMessageRetrieverDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="ChatMessageRetrieverDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The chat message retriever dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace Services.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The chat message retriever dto.
    /// </summary>
    [DataContract]
    public class ChatMessageRetrieverDTO
    {
        /// <summary>
        /// Gets or sets the sender id.
        /// </summary>
        [DataMember]
        public int SenderId { get; set; }

        /// <summary>
        /// Gets or sets the recipient id.
        /// </summary>
        [DataMember]
        public int RecipientId { get; set; }

        /// <summary>
        /// Gets or sets the load range dto.
        /// </summary>
        [DataMember]
        public LoadRangeDTO LoadRangeDTO { get; set; }
    }
}